index.js

```diff
@@ -8,18 +8,23 @@ const ObjectId = MongoDb.ObjectId
 
 function fastifyMongodb (fastify, options, next) {
   if (options.client) {
-    const db = options.client
-    delete options.client
+    const client = options.client
+
+    const databaseName = options.database
+
     const mongo = {
-      db: db,
-      ObjectId: ObjectId
+      client: client,
+      ObjectId: ObjectId,
+      dbs: {}
     }
     if (options.name) {
       mongo[options.name] = mongo
-      delete options.name
+    }
+    if (databaseName) {
+      mongo.dbs[databaseName] = client.db(databaseName)
```

Comment:
I think this is redundant, we already have `mongo[name] = mongo`

Reply:
I don't understand what you mean.

The `name` parameter identifies a mongodb cluster, `database` parameter identifies a database on that cluster. Potentially you can have multiple databases assigned inside `mongo.dbs` (currently not implemented)

Reply:
The basic case is that you connect a single database. I see the value in providing `mongo.db` as it was before. Can you cover that case?